docs/tooling/avalanche-cli.md

```diff
@@ -137,14 +137,14 @@ avalanche subnet removeValidator [subnetName] [flags]
 
 ### Subnet Change Owner
 
-The `subnet changeOwner` changes the owner of the deployed Subnet.
+The `blockchain changeOwner` changes the owner of the deployed Subnet.
 
 This command currently only works on Subnets deployed to Devnet, Fuji or Mainnet.
```

Comment:
https://github.com/ava-labs/avalanche-cli/pull/2063/files already changed all helps (cmd.Long). Maybe you can copy those contents into here, for each command.